wavefront/resource_metrics_policy.go

```diff
@@ -0,0 +1,360 @@
+package wavefront
+
+import (
+	"errors"
+	"fmt"
+	"github.com/WavefrontHQ/go-wavefront-management-api"
+	"github.com/hashicorp/go-cty/cty"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+const (
+	updaterIdKey          = "updater_id"
+	updatedEpochMillisKey = "updated_epoch_millis"
+	accountsKey           = "account_ids"
+	tagsKey               = "tags"
+	prefixesKey           = "prefixes"
+	tagsAndedKey          = "tags_anded"
+	accessTypeKey         = "access_type"
+	userGroupsKey         = "user_group_ids"
+	policyRulesKey        = "policy_rules"
+	policyTagKey          = "key"
+	policyTagValue        = "value"
+	roleIdsTagKey         = "role_ids"
+)
+
+func resourceMetricsPolicy() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceMetricsPolicyUpdate,
+		Read:   resourceMetricsPolicyRead,
+		Update: resourceMetricsPolicyUpdate,
+		Delete: resourceMetricsPolicyDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Schema: resourceMetricsPolicySchema(),
+	}
+}
+
+func resourceMetricsPolicyRead(d *schema.ResourceData, meta interface{}) error {
+	metrics := meta.(*wavefrontClient).client.MetricsPolicyAPI()
+	metricsPolicy, err := metrics.Get()
+	if err != nil {
+		return fmt.Errorf("error retrieving metrics policy: %d", err)
+	}
+	d.SetId(string(rune(metricsPolicy.UpdatedEpochMillis)))
```

Comment:
Why does this work?  A rune is 4 bytes.  We are already past 1.6 trillion seconds since Jan 1 1970, and 1.6 trillion can't fit into 4 bytes.  Would it make more sense to make the Id be the string value of UpdatedEpochMillis?

Reply:
Updating to use strconv.Itoa() for int to string conv